distribution/sdk/sample-mpegts-stream-generator/src/main/java/org/codice/alliance/distribution/sdk/video/stream/mpegts/MpegTsUdpClient.java

```diff
@@ -0,0 +1,277 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.distribution.sdk.video.stream.mpegts;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecuteResultHandler;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.ExecuteWatchdog;
+import org.apache.commons.exec.Executor;
+import org.apache.commons.exec.PumpStreamHandler;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.netty.bootstrap.Bootstrap;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.SimpleChannelInboundHandler;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.DatagramPacket;
+import io.netty.channel.socket.nio.NioDatagramChannel;
+
+/**
+ * This client is used for testing/development to transmit an MPEG-TS file as a stream of UDP
+ * packets.
+ */
+public class MpegTsUdpClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MpegTsUdpClient.class);
+
+    private static final int PACKET_SIZE = 188;
+
+    private static final String DEFAULT_IP = "127.0.0.1";
+
+    private static final int DEFAULT_PORT = 50000;
+
+    private static final String SUPPRESS_PRINTING_BANNER_FLAG = "-hide_banner";
+
+    private static final String INPUT_FILE_FLAG = "-i";
+
+    private static final boolean DONT_HANDLE_QUOTING = false;
```

Comment:
Recommend changing this to `HANDLE_QUOTING`. 
Don't not avoid double negative logic. I get too confused. :)